services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx

```diff
@@ -235,7 +235,6 @@ export const EntityDetailsOverlay = ({
           initiativeName={reportPageTitle}
         />
       )}
-      {verbiage.intro.title && <Box sx={sx.title}>{verbiage.intro.title}</Box>}
```

Comment:
Removing this prop and instead putting the heading content in the `info` custom elements array